resdk/resources/permissions.py

```diff
@@ -269,3 +276,43 @@ def __repr__(self):
                 ))
 
         return '\n'.join(res)
+
+    def _get_holders_with_perm(self, perm):
+        """Get Users/Group/public names that have ``perm`` perm."""
+        # Prevent circular imports
+        from .user import Group, User
+
+        self.fetch()
+
+        holders = []
+        for item in self._permissions:  # pylint: disable=protected-access
+            if perm in item['permissions']:
```

Comment:
You can use [filter](https://book.pythontips.com/en/latest/map_filter.html#filter) instead of this if.

Reply:
I believe this approach is clearer than using filter, but this might be a matter of taste.